index.html

```diff
@@ -5,11 +5,375 @@
+      <nav class="nav">
+        <h5 class="nav__logo">Medical Care</h5>
+        <ul class="nav-list nav--default">
+          <li class="nav__item"><a href="/#" class="nav__link">Home</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Products</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Pricing</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="header-action">
+        <button class="button button--bg-white nav--default">Login</button>
+        <button
+          class="button button--bg-green nav--default button--after-icon-arrow"
+        >
+          Join Us
+        </button>
+        <button class="nav--mobile header-action__menu-icon">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            class="w-6 h-6"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+            />
+          </svg>
+        </button>
+      </div>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="hero__content">
+          <h3 class="title">Welcome</h3>
+          <h5 class="headline headline--size-large">
+            A Great Place <br />
+            to Receive Care
+          </h5>
+          <p class="description description--size-large">
+            Get your best looking smile now!
+          </p>
+          <div class="hero-action">
+            <button class="button button--bg-green">Join Us</button>
+            <button class="button button--bg-white button--border-green">
+              Learn More
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="our-experts">
+        <img src="/images/col-md-6.png" alt="expert" class="our-experts__img" />
+
+        <div class="our-experts__content">
+          <h2 class="headline headline--size-medium headline--before-line">
+            Meet our Experts
+          </h2>
+          <p class="description description--size-medium">
+            Problems trying to resolve the conflict between the two major realms
+            of classical physics: Newtonian mechanics
+          </p>
+          <button class="button button--bg-white button--after-icon-chevron">
+            Learn more
+          </button>
+        </div>
+      </section>
+      <section class="team">
+        <h2 class="title">Team</h2>
+        <h2 class="headline headline--size-medium">Our Team</h2>
+        <p class="description description--size-medium descrition--width-half">
+          Problems trying to resolve the conflict between the two major realms
+          of classical physics: Newtonian mechanics
+        </p>
+        <div class="team-members">
```

Comment:
Why don't you make `team-members` an element of `team` block and let it contain `card` blocks. If you insist on making `team-members` a block, you should give it a different name like `members` so that you don't  semantically tie it to `team` block and make it reusable.